pkg/apis/navigator/v1alpha1/types.go

```diff
@@ -77,10 +97,13 @@ type CassandraClusterNodePool struct {
 }
 
 type CassandraClusterStatus struct {
-	NodePools map[string]CassandraClusterNodePoolStatus `json:"nodePools"`
+	// The status of each node pool in the CassandraCluster.
+	// This will be periodically updated by Navigator.
+	NodePools map[string]CassandraClusterNodePoolStatus `json:"nodePools,omitempty"`
```

Comment:
Same comment as for `Spec.NodePools` above. I think I'd prefer this to be shown as an empty map in the serialized json than omitted.

Reply:
That's not possible given how maps are serialised in Go. This field is also optional, as it may not always be set, hence it needs omitempty (and should also have the +optional tag)